.github/workflows/release.yaml

```diff
@@ -0,0 +1,131 @@
+on:
+  push:
+    tags:
+      - 'litani-*'
+
+name: Create Release and package
+
+jobs:
+  get-version-information:
+    name: Get Version Information
+    runs-on: ubuntu-20.04
+    outputs:
+      version: ${{ steps.split-version.outputs._1 }}
+    steps:
+      - uses: jungwinter/split@v1
+        id: split
+        with:
+          msg: ${{ github.ref }}
+          seperator: '/'
+      - uses: jungwinter/split@v1
+        id: split-version
+        with:
+          msg: ${{ steps.split.outputs._2 }}
+          seperator: '-'
+  perform-release:
+    name: Perform Release
+    runs-on: ubuntu-20.04
+    needs: get-version-information
+    outputs:
+      version: ${{ needs.get-version-information.outputs.version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          LITANI_VERSION: ${{ needs.get-version-information.outputs.version }}
+        with:
+          tag_name: litani-${{ env.LITANI_VERSION }}
+          release_name: litani-${{ env.LITANI_VERSION }}
+          body: |
+            This is Litani version ${{ env.LITANI_VERSION }}.
+
+            ## MacOS
+
+            On MacOS, install Litani using [Homebrew](https://brew.sh/) with
+
+            ```sh
+            brew tap aws/tap
+            brew install litani
+            ```
+
+            or upgrade (if it's already been installed) with:
+
+            ```sh
+            brew upgrade litani
+            ```
+
+            ## Ubuntu
+
+            On Ubuntu, install Litani by downloading the *.deb package below for Ubuntu 20 and install with
+
+            ```sh
+            # Ubuntu 20.04:
+            $ apt install -y ./litani-${{ env.LITANI_VERSION }}.deb
+            ```
+          draft: false
+          prerelease: false
+  ubuntu-20_04-package:
+      runs-on: ubuntu-20.04
+      needs: perform-release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        LITANI_VERSION: ${{ needs.perform-release.outputs.version }}
+      steps:
+        - uses: actions/checkout@v2
+        - name: Setup directory for deb package
+          run: |
+            sudo apt-get install -y mandoc scdoc ninja-build
+            mkdir -p litani-${{ env.LITANI_VERSION }}/{DEBIAN,usr/{bin,libexec/litani,share/{doc/litani,man/{man1,man5,man7}}}}
+            touch litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            cat << EOF > litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            Package: Litani
+            Version: ${{ env.LITANI_VERSION }}
+            Architecture: amd64
+            Depends: ninja-build, gnuplot, graphviz, python3-jinja2, python3-voluptuous
```

Comment:
Let's not install voluptuous. I'd like to keep this as optional.

Reply:
Resolved